commands/copy_paste.go

```diff
@@ -37,11 +38,15 @@ func (c *Copy) Exec(target interface{}) (executed, consume bool) {
 	if editor == nil {
 		return true, true
 	}
-	if editor.Controller().SelectionCount() != 1 {
-		log.Print("Warning: copy can currently only copy the first selection")
+
+	selections := editor.Controller().Selections()
+	var buffer bytes.Buffer
+	for i := 0; i < selections.Len(); i++ {
+		buffer.WriteString(editor.Controller().SelectionText(i))
+		buffer.WriteString("\n")
```

Comment:
This is going to add a newline to any copied text (even if you're only copying a single selection).  I think this needs to change in one of the following ways:

A) Remove this line and assume that multiple selected blocks of text should be concatenated
or
B) Skip this line if `i == selections.Len() - 1`

I think I would prefer option (A), personally.  If I'm selecting multiple things and copying them, I think I would prefer to not have a newline between each of my selections.